### PR TITLE
Fix build, clean on fresh install

### PR DESCRIPTION
**Pull request by egarson**
This change fixes building the project on fresh install, by specifying to install lerna globally and respects caching within node_modules by not nuking `node_modules/`. This was causing a subtle chicken-and-egg problem on a fresh install.

It "fixes" Installation and integration test instructions broken? #2134.

This is almost entirely documentation-related with the notable exception of a single character in package.json, to delete the contents of `node_modules/` but not the directory itself (and more importantly it's hidden `nx` cache within it).

I also took the opportunity to add instructions to run the integration tests via flextesa :tada:.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,29 +58,29 @@ While other versions often work, the above are what we officially support. YMMV!
...
 We are active and enthusiastic participants of the following community support channels:
 
-- [TezosTaquito Telegram Community Channel][telegram]
+- [ECAD Labs Discord Channel][discord]
 - [Tezos StackExchange][stackexchange]
 
 ## Project Organization
 
-`taquito` is organized as a [mono repo](https://en.wikipedia.org/wiki/Monorepo), and includes several npm packages published to npmjs.org under the `@taquito` handle. Each package has its own README and can be found in the `packages/` directory
+Taquito is organized as a [monorepo](https://en.wikipedia.org/wiki/Monorepo), and is compose of several npm packages that are [published to npmjs.org](https://www.npmjs.com/package/@taquito/taquito) under the `@taquito` handle. Each package has its own README which can be found in the corresponding directory within `packages/`.
```
Comment (roxaneletourneau) [inline README.md:66]: compose -> composed

File: README.md [hunk 2]
```diff
@@ -58,29 +58,29 @@ While other versions often work, the above are what we officially support. YMMV!
...
-| [@taquito/signer](packages/taquito-signer)                       | Provide necessary function to sign using tezos keys           |
-| [@taquito/utils](packages/taquito-utils)                         | Provide different encoding and decoding utilities             |
-| [@taquito/tzip12](packages/taquito-tzip12)                       | TZIP-12 Implementation for Taquito                            |
-| [@taquito/tzip16](packages/taquito-tzip16)                       | TZIP-16 Implementation for Taquito                            |
-| [@taquito/beacon-wallet](packages/taquito-beacon-wallet)         | TZIP-10 Wallet Interaction implementation for the Wallet API  |
+| [@taquito/signer](packages/taquito-signer)                       | Provides functionality to sign data using tezos keys          |
+| [@taquito/utils](packages/taquito-utils)                         | Provides different encoding and decoding utilities            |
+| [@taquito/tzip12](packages/taquito-tzip12)                       | TZIP-12 implementation for Taquito, related to NFTs           |
```
Comment (roxaneletourneau) [inline README.md:81]: I wonder if "related to NFTs" is too vague. The current implementation of tzip 12 and 1zip16 only relate to metadata.

tzip 12 allows retrieving token metadata
tzip 16 allows retrieving contract metadata and executing off-chain views.

File: README.md [hunk 3]
```diff
@@ -58,29 +58,29 @@ While other versions often work, the above are what we officially support. YMMV!
...
 | -------------------------------------------------------------- | ------------------------------------------------------------ |
 | [@taquito/taquito](packages/taquito)                           | [Facade](https://en.wikipedia.org/wiki/Facade_pattern) to lower-level, package-specific functionality  |
 
 | Low-Level Packages                                               | Responsibility                                                |
 | ---------------------------------------------------------------- | ------------------------------------------------------------- |
-| [@taquito/local-forging](packages/taquito-local-forging)         | Local "forging" of Tezos operations                           |
-| [@taquito/michelson-encoder](packages/taquito-michelson-encoder) | Creates a JS abstraction for Smart Contracts                  |
-| [@taquito/michel-codec](packages/taquito-michel-codec)           | Converts Michelson between forms and expands Macros           |
+| [@taquito/local-forging](packages/taquito-local-forging)         | Local "forging" (preparation) of Tezos operations             |
```
Comment (roxaneletourneau) [inline README.md:74]: preparation -> serialization of Tezos operations into bytes?

File: README.md [hunk 4]
```diff
@@ -156,25 +173,50 @@ See the top-level `package.json` "scripts" section. Some common targets are:
 
 The Taquito integration tests are located in the `integration-tests/` directory.
 
-To run the integration tests, `cd` into `integration-tests/` and run `npm run test` (ensure you have completed the build steps as described earlier in this README file.). The integration test suite runs all tests against the current tezos protocol testnet, and typically also against the previous and next protocol testnets. See the `scripts` property in the `integration-tests/package.json` file for specific test targets.
+To prepare to run the integration tests, perform the following steps:
```
Comment (roxaneletourneau) [inline README.md:176]: These instructions are for running the integration tests with flextesa, but it is also possible to run them with a public node. There are instructions in the integration-tests/README.md file as well to run the integration tests. Should everything be consolidated in one place?

File: README.md (hunk 1 above)
Comment (egarson) [reply README.md:66]: Good catch! :+1: 
I will *not* be correcting your French anytime soon :joy:.

File: README.md (hunk 2 above)
Comment (egarson) [reply README.md:81]: Thank you - I was hand-waving and this is far better.

File: README.md (hunk 3 above)
Comment (egarson) [reply README.md:74]: Yes, better and more clear.

File: README.md (hunk 4 above)
Comment (egarson) [reply README.md:176]: Woops, I "forgot" about the README in `integration-tests`.
Yes, this should be consolidated there. We should provide instructions to run against both environments.
I will add a placeholder in the top-level README to go look at `integration-tests/README` to learn how to run them. I will move the instructions @danielelisi kindly helped me with to that location at the same time.

**Review by roxaneletourneau — APPROVED** [review]

**Review by danielelisi — APPROVED** [review]
LGTM 👍

**Pull request merged**